public/app/features/expressions/components/Math.tsx

```diff
@@ -36,93 +34,97 @@ export const Math: FC<Props> = ({ labelWidth, onChange, query, onRunQuery }) =>
   return (
     <Stack direction="row">
       <InlineField
-        label="Expression"
```

Comment:
💬 The changes in this file are mostly to refactor the `Math` expression help text, to support a narrower card view I moved the info to a hover card that is shown when you hover over the "Expression (?)" label. This also works nicer with the panel view (where we also support expressions).